src/renderer/component/video/internal/player.jsx

```diff
@@ -121,6 +124,11 @@ class VideoPlayer extends React.PureComponent {
     }
   }
 
+  pauseVideo() {
```

Comment:
what if you change this to `componentWillReceiveProps` instead of `pauseVideo`?

Reply:
then of course you would need to filter out the prop changes that don't lead to a pause, but does that make sense?

Reply:
I tried that and the method(?) was never called.

I think the issue is that the selector isn't working, or that the reducer isn't actually changing the state. My focus is on line 63 of components/video.view.jsx where it console logs the value of videoPause. It's always false, even after I've clicked open, which should update valuePause to true through the reducer.

Reply:
@daovist try using the console to help debug what unexpected behavior is happening here. The console will show all action dispatches as well as all state changes (expand the console messages that are printed on action dispatches). This should tell you whether it's an issue with actions or the reducer. If both the action is fired and the Redux state looks correct, you can add a console statement to see if the selector is firing when you think it ought to.